src/dmclock_server.h

```diff
@@ -1522,10 +1533,11 @@ namespace crimson {
 			std::chrono::duration<Rep,Per> _erase_age,
 			std::chrono::duration<Rep,Per> _check_time,
 			AtLimitParam at_limit_param = AtLimit::Wait,
-			double anticipation_timeout = 0.0) :
+			double anticipation_timeout = 0.0,
+			Counter _erase_max = 100) :
```

Comment:
Do you foresee users wanting to change this? If not, why don't we move this to after _check_time and make it fixed as well. I realize that it will require coordinated changes with ceph itself when it updates the version of dmclock that it uses, but I'd rather be consistent than convenient.

Reply:
yeah, I originally put `_erase_max` here as an argument to make it convenient to change as needed, then I add a configurable item and pass it in from ceph, but I'm not realize that it brings compatibility problems since dmclock has been change from subtree to submodule in ceph (we are Luminous), so I'll drop it and make it fixed.